entity-builder.py

```diff
@@ -40,7 +40,7 @@
 # it would be nice to have this request settings if the gateway isn't the remote node
 mqtt.add_argument(
     "--root-topic",
-    help="The root topic to use in MQTT for the generated files. If not provided, will attempt to get the root path from the local node and use LongFast as the channel.",
+    help="The root topic to use in MQTT for the generated files. If not provided, will attempt to get the root path from the local node and use `msh/LongFast` as the channel. Wildcard: `+`. Example: to include all channels with the root topic `msh/`, use `msh/2/json/+`.",
```

Comment:
I think this isn't quite right -- it's not `msh/LongFast` it's using as the channel, it is `LongFast`. But there's semantic confusion between channels and MQTT topics and I'm not sure how to resolve it, heh. I guess maybe we should explain what the root topic actually does, possibly? The structure is: the root topic that's configured on the gateway, plus the fixed string `/2/json/`, plus either a channel name to use or a `+` to denote all channels. So the default here, if the local node is configured with `msh` as the root topic, it will produce `msh/2/json/LongFast`.

I guess the other way we could go with this is to more closely match how nodes are configured and just change the arguments this script takes. I guess then we'd change it so `--root-topic` is expected to match the gateway node (so just `msh` or `msh/US` or whatever, pulled directly from the connected node if not provided), plus an added argument `--channel` that denotes a channel to use, with the default being `+` so it listens on all channels. Then users wouldn't need to know about the magic `/2/json/` in the middle at all, which I suppose is probably good too.

Reply:
I've removed msh/ from the channel name.